### ISSUE TITLE
Why is the output of the model has no grads?

### ISSUE THREAD

**Issue by wzjin2017** [eriklindernoren/PyTorch-YOLOv3#260]
Hi, thank you for building this pytorch version of YOLO v3. I am implementing my codes on the basis of this project. I used the darknet model in this project and the output's requires_grad is False. How can I make the output coming out the model has grad data? Could you point me to the place where I should look and modify?

**Comment by zihanzhang1997** [comment]
I have met the same problem. Have you solved this?

**Comment by Flova** [comment]
Are you in test or training mode? The model is not differentiable in test mode as it directly outputs a set of boxes.
